app/views/charities/_charities.html.erb

```diff
@@ -1,8 +0,0 @@
-<h1>Charity Results</h1>
-  <% if @charities.present? %>
-    <% @charities.each do |x| %>
-      <p><%= link_to x.name, charity_path(x.id) %></p>
-    <% end %>
-  <% else %>
-    <p><%= t('.search_failure') %></p>
- <% end %>
```

Comment:
_charities.html.erb was a partial view I used solely to display the pre-Sunspot-gem search. Since Sunspot implements search, I instead created the Search controller and related search view.